test/bigtest/helpers/PluginHarnessSingleSelect.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import noop from 'lodash/noop';
+
+import { Pluggable } from '@folio/stripes/core';
+
+const PluginHarnessSingleSelect = props => (
+  <Pluggable
+    aria-haspopup="true"
+    type="find-import-profile"
+    id="clickable-find-import-profile"
+    searchLabel="Choose Profile"
+    marginTop0
+    searchButtonStyle="link"
+    addLines={noop}
+    isSingleSelect
+    {...props}
+  >
+    <span data-test-no-plugin-available>No plugin available!</span>
```

Comment:
I would suggest translating this as well. I see in [ui-acquisition-units](https://github.com/folio-org/ui-acquisition-units/blob/b090280388b25d3b51c72da31f22790624edbedf/src/settings/AcquisitionUnits/AcquisitionUnitMemberships/AssignAcquisitionUnitMemberships.js#L26) implemented that way. I also faced cases without it, but I think it would be better to have strings translated when possible.

Reply:
Done